### ISSUE TITLE
abnormal susceptibility to glomerulosclerosis

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3665]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1.  abnormal susceptibility to glomerulosclerosis
2.  decreased susceptibility to glomerulosclerosis
3.  increased susceptibility to glomerulosclerosis

>2. Synonyms (e.g., absent spleen)

1.  abnormal susceptibility to glomerular nephritis
2.  decreased susceptibility to glomerular nephritis
3.  increased susceptibility to glomerular nephritis

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1.  a change in the likelihood or extent of inflammation and scarring damage to the glomerulus usually resulting from an autoimmune response.
2.  a diminished likelihood or extent of inflammation and scarring damage to the glomerulus usually resulting from an autoimmune response.
3.  an increased likelihood or extent of inflammation and scarring damage to the glomerulus usually resulting from an autoimmune response.

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:20488952

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1.  glomerulonephritis (MP:0002743)
2.  abnormal susceptibility to glomerulosclerosis (new term)
3.  abnormal susceptibility to glomerulosclerosis (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0003-3645-6803

**Comment by sbello** [comment]
abnormal susceptibility to glomerulonephritis - MP:0021199
increased susceptibility to glomerulonephritis - MP:0021201
decreased susceptibility to glomerulonephritis - MP:0021200

entered as glomerulonephritis since the synonyms and definitions match this term and not glomerulosclerosis
